swagger.yaml

```diff
@@ -4118,7 +4118,7 @@ components:
           format: uuid
           readOnly: true
         name:
```

Comment:
Maybe rename this to localizedNames and change the type to be array which contains list of LocalizedValue objects. This way we don't need that extra LocalizedName object in between.

Reply:
Resolved. Originally I used the extra object since it's going to be reused a lot.